dags/vz_populate_cr3_metadata.py

```diff
@@ -0,0 +1,81 @@
+import os
+from pendulum import datetime, duration
+from airflow.decorators import dag, task
+from airflow.operators.docker_operator import DockerOperator
+from utils.slack_operator import task_fail_slack_alert
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "execution_timeout": duration(minutes=5),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+REQUIRED_SECRETS = {
+    "AWS_BUCKET_NAME": {
+        "opitem": "Vision Zero CR3 Metadata Extraction",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Bucket for CR3s",
+    },
+    "AWS_BUCKET_ENVIRONMENT": {
+        "opitem": "Vision Zero CR3 Metadata Extraction",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS S3 Path for CR3s",
+    },
+    "HASURA_ENDPOINT": {
+        "opitem": "Vision Zero graphql-engine Endpoints",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.GraphQL Endpoint",
+    },
+    "HASURA_ADMIN_KEY": {
+        "opitem": "Vision Zero graphql-engine Endpoints",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Admin Key",
+    },
+    "AWS_ACCESS_KEY_ID": {
+        "opitem": "Vision Zero CRIS Import",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Access key",
+    },
+    "AWS_SECRET_ACCESS_KEY": {
+        "opitem": "Vision Zero CRIS Import",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Secret key",
+    },
+}
+
+
+@dag(
+    dag_id="vz-populate-cr3-metadata",
+    description="A DAG which populates CR3 metadata in the VZDB",
```

Comment:
I wanted to link to [my comment from the corresponding PR](https://github.com/cityofaustin/atd-vz-data/pull/1328#discussion_r1432895861) because I'm still wondering how we can avoid a situation in the future where we forget how important this metadata is to the diagram and narrative extraction. I'm 100% talking about a future enhancement. This works great to get us back up and running. 🙌 🙏